contrib/guix/guix-attest

```diff
@@ -174,6 +174,10 @@ basenameify_SHA256SUMS() {
     sed -E 's@(^[[:xdigit:]]{64}[[:space:]]+).+/([^/]+$)@\1\2@'
 }
 
+unix2dos() {
```

Comment:
Maybe refer to https://www.rfc-editor.org/rfc/rfc4880.html section 5.2.4 here.

Reply:
Done